test/e2e/operator_test.go

```diff
@@ -0,0 +1,290 @@
+//+build e2e
```

Comment:
These tests seem to be quite OpenShift oriented (infrastructure config, aws credentials secret). Maybe it makes sense to tag this as openshift e2e.